components/call_button/call_button.jsx

```diff
@@ -0,0 +1,134 @@
+            bodyAction = (
+                <div>
+                    <button
+                        type='button'
+                        id='calldButton'
+                        className='style--none post-action icon icon--attachment'
+                        onClick={() => {
+                            if (item.action) {
+                                item.action(this.props.currentChannel, this.props.channelMember);
+                            }
+                        }}
+                        onTouchEnd={() => {
+                            if (item.action) {
+                                item.action(this.props.currentChannel, this.props.channelMember);
+                            }
+                        }}
+                    >
+                        {item.icon}
+                    </button>
+                </div>
+            );
+        } else {
+            const pluginFileUploadMethods = this.props.pluginCallMethods.map((item) => {
+                return (
+                    <li
+                        key={item.pluginId + item.id + '_callbuttonpluginmenuitem'}
+                        onClick={() => {
+                            if (item.action) {
+                                item.action(this.props.currentChannel, this.props.channelMember);
+                            }
+                            this.setState({menuOpen: false});
+                        }}
+                    >
+                        <a href='#'>
+                            <span className='mr-2'>
+                                {item.icon}
+                            </span>
+                            {item.dropdownText}
+                        </a>
+                    </li>
+                );
+            });
+            bodyAction = (
+                <div>
+                    <MenuWrapper>
+                        <button
+                            type='button'
+                            className='style--none post-action'
+                        >
+                            <div
+                                id='callButton'
+                                className='icon icon--attachment'
+                            >
+                                <CameraIcon className='d-flex'/>
+                            </div>
+                        </button>
+                        <Menu
+                            id='callOptions'
+                            openLeft={true}
+                            openUp={true}
+                            ariaLabel={formatMessage({id: 'call_button.menuAriaLabel', defaultMessage: 'Call type selector'})}
+                            customStyles={customStyles}
+                        >
+                            {pluginFileUploadMethods}
+                        </Menu>
+                    </MenuWrapper>
+                </div>
+            );
+        }
+
+        return (
+            <div className={'style--none'}>
```

Comment:
What does the class `style--none` do here? I think it might only be necessary on `button` elements, not divs. If that's the case, we can also just `return bodyAction;` here and remove the extra div.

Reply:
Not sure at the moment what is the use, again copy pasted from FileUpload. I will double check.